packages/ibm-products/src/components/OptionsTile/OptionsTile.tsx

```diff
@@ -26,60 +21,132 @@ import {
   WarningFilled,
 } from '@carbon/react/icons';
 import * as carbonMotion from '@carbon/motion';
+import { CarbonIconType } from '@carbon/icons-react/lib/CarbonIcon';
 
-// The block part of our conventional BEM class names (blockClass__E--M).
 const blockClass = `${pkg.prefix}--options-tile`;
 const componentName = 'OptionsTile';
 
-// NOTE: the component SCSS is not imported here: it is rolled up separately.
+interface OptionsTileProps {
+  /**
+   * Provide content to render as expandable OptionsTile. If no children
+   * are present, the OptionsTile will render as its variant.
+   */
+  children?: ReactNode;
 
-// Default values for props
-const defaults = {
-  onChange: () => {},
-  size: 'xl',
-};
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className?: string;
+
+  /**
+   * Whether the toggle is enabled or disabled. If nothing is passed,
+   * no toggle will be rendered.
+   */
+  enabled?: boolean;
+
+  /**
+   * Whether the OptionsTile is in invalid validation state.
+   */
+  invalid?: boolean;
+
+  /**
+   * Provide a text explaining why the OptionsTile is in invalid state.
+   */
+  invalidText?: string;
+
+  /**
+   * Whether the OptionsTile is in locked validation state.
+   */
+  locked?: boolean;
+
+  /**
+   * Provide a text explaining why the OptionsTile is in locked state.
+   */
+  lockedText?: string;
+
+  /**
+   * Provide a function which will be called each time the user
+   * toggles the open state of the OptionsTile.
+   */
+  onChange?: (value: any) => void;
+
+  /**
+   * Provide a function which will be called each time the user
+   * interacts with the toggle.
+   */
+  onToggle: (value: any) => void;
+
+  /**
+   * Whether the OptionsTile is in open state.
+   */
+  open?: boolean;
+
+  /**
+   * Define the size of the OptionsTile.
+   */
+  size: 'lg' | 'xl';
+
+  /**
+   * Optionally provide a text summarizing the current state of the content.
+   */
+  summary?: string;
+
+  /**
+   * Provide the title for this OptionsTile.
+   */
+  title: string;
+
+  /**
+   * Optionally provide an id which should be used for the title.
+   */
+  titleId?: string;
+
+  /**
+   * Whether the OptionsTile is in warning validation state.
+   */
+  warn?: boolean;
+
+  /**
+   * Provide a text explaining why the OptionsTile is in warning state.
+   */
+  warnText?: string;
+}
 
-/**
- * TODO: A description of the component.
- */
 export let OptionsTile = React.forwardRef(
   (
     {
-      // The component props, in alphabetical order (for consistency).
-
       children,
       className,
       enabled,
       invalid,
       invalidText,
       locked,
       lockedText,
-      onChange = defaults.onChange,
+      onChange,
       onToggle,
       open,
-      size = defaults.size,
+      size = 'lg',
```

Comment:
Are we able to revert back to the previous approach with default props with the `defaults` object? I also accidentally removed this in the `SidePanel` PR but you should be able to add it back without problems 🤞